### PR TITLE
libgetdata 0.9.2.1

### PR DESCRIPTION
**Pull request by ilovezfs**
Also, fix sandbox violation when building with the perl option.

### CODE REVIEW

File: Formula/libgetdata.rb [hunk 1]
```diff
@@ -12,14 +12,14 @@ class Libgetdata < Formula
   end
 
   option "with-fortran", "Build Fortran 77 bindings"
-  option "with-perl", "Build Perl binding"
```
Comment (UniqMartin) [inline Formula/libgetdata.rb]: Maybe keep the option nonetheless so that it is clearer what type of Perl support this formula has to offer?

Comment (ilovezfs) [inline Formula/libgetdata.rb]: Doesn't seem necessary. What other kind of perl support is there besides building a pm for random program foo?

Comment (UniqMartin) [inline Formula/libgetdata.rb]: Something that uses Perl as opposed to providing an interface to itself from Perl …

Comment (ilovezfs) [inline Formula/libgetdata.rb]: If it were something that uses perl, would perl be optional or required?

Comment (ilovezfs) [inline Formula/libgetdata.rb]: @UniqMartin I sympathize with what you're saying, but it seems like the meaning of a perl *option* is obvious, so if anything your concern would suggest the default wording of :perl => :optional/:recommended (or, for that matter, :python => :optional/:recommended) should be changed if it's not sufficiently clear what building with perl support or building with python support means without further customized option descriptions.

Comment (UniqMartin) [inline Formula/libgetdata.rb]: I guess that depends. Anyway, just wanted to check. I'm not feeling overly passionate about this option, thus feel free to keep the PR as-is.

Comment (ilovezfs) [inline Formula/libgetdata.rb]: I'm actually happy to change it back if you'd like. Just thought I'd give @DomT4's cool new :perl thingy and spin, and see if we could "get away with" the default option wording it provides.

**Comment by UniqMartin** [comment]
:+1:

**Comment by ilovezfs** [comment]
@DomT4 any thoughts on this use of `:perl => optional` vs. providing a customized option description? It would be good, I think, to have a "policy" concerning perl/python "bindings" and whether the default descriptions suffice, or if they should be customized, since optional bindings are a pretty common thing.

File: Formula/libgetdata.rb (hunk 1 above)
Comment (DomT4) [inline Formula/libgetdata.rb]: IMO it's worth introducing the `:perl` dependency _where_ it's optional anyway. It makes sense to build bindings against the version of Perl actually first in the user's `$PATH`, providing that's a suitable version of Perl to meet the requirement.

The only area we need to be particularly wary is where it isn't `:optional`, given that that drags in the need to install our `perl` (as is the situation with `python` as well).

**Comment by DomT4** [comment]
Left a nested comment. Generally we do permit `option` usage on stuff like `:python` and `:perl` to highlight building with those is useful for bindings, when that is the case. This hasn't been applied consistently, though.

File: Formula/libgetdata.rb (hunk 1 above)
Comment (ilovezfs) [inline Formula/libgetdata.rb]: OK, I'll put ""Build Perl binding" back in then

**Comment by DomT4** [comment]
👍

**Pull request closed without merging**